crates/ide_diagnostics/src/handlers/missing_fields.rs

```diff
@@ -167,7 +167,7 @@ fn main() {
 pub struct Foo { pub a: i32, pub b: i32 }
 "#,
             r#"
-fn some(, b: () ) {}
+fn some(, b: Default::default() ) {}
```

Comment:
This is wrong.

Reply:
I think that's what the comment above says: `FIXME: this is very wrong, but somewhat tricky to fix.`

Reply:
Ye that's the macro at fault, nothing specific to this PR